src/I8Beef.UniFi.Video/Protocol/Bootstrap/Bootstrap.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using I8Beef.UniFi.Video.Protocol.Common;
+
+namespace I8Beef.UniFi.Video.Protocol.Bootstrap
+{
+    /// <summary>
+    /// Bootstrap response.
+    /// </summary>
+    public class Bootstrap
+    {
+        /// <summary>
+        /// Admin user group id.
+        /// </summary>
+        public string AdminUserGroupId { get; set; }
+
+        // todo: alertSchedules
```

Comment:
Down in the Servers object, you added a few "dynamic" properties, I'm assuming because like mine, you got "null" for those in the response. I don't have alert schedules or camera schedules to know what these two structures should be, but might it make sense to actually put in IList<dynamic> on these two to follow the approach of "We don't know what these are, so we don't have real objects for them, but at least you have something to work with"?